omero/sysadmins/python3-migration.rst

```diff
@@ -0,0 +1,128 @@
+Migration to Python 3
+=====================
+
+.. warning::
+
+  This is still a pre-release of OMERO 5.6. The instructions below
+  are intended to help plan a migration path from your current OMERO
+  installation. We suggest you **not** perform the migration on your
+  production systems until the 5.6.0 release is ready.
+
+Basic steps
+-----------
+
+1. Choose a platform and a python. If your current installation platform
+   does not match one of the :doc:`recommended platforms <version-requirements>`,
+   you may want to choose a new platform as your migration target. See
+   `Choosing a platform`_ below.
+2. Install OMERO.server and OMERO.web *separately*. Though not necessary, all
+   instructions like `OMERO.server`_ and `OMERO.web`_ below as well as the main
+   :doc:`server <unix/server-installation>` and 
+   :doc:`web <unix/install-web/web-deployment>` installation pages now assume
+   that the two are in separate installations.
+3. Once both have been installed, perform a
+   :doc:`backup and restore <server-backup-and-restore>` procedure
+   and test your installation against the copy of your data.
+
+Choosing a platform
+-------------------
+
+We suggest Python 3.6. This is the default installation on the two recommended platforms,
```

Comment:
First sentence should say something about platform. Instead, it starts talking about python version. Slightly confusing considering the header of this section.

Maybe 
Whatever platform you finally choose, we would suggest you use Python 3.6.